### DISCUSSION TITLE
How can i generate dimension and datasets tables

### DISCUSSION THREAD

**Discussion by GitVivekHub** [Sunbird-cQube/community#786] [General]
I am trying to upload the CSV for national programs ( eg DIKSHA-ETB )  vie national/programs API for cQube . I am able to load the 

CSV to the emission-folder in the minio bucket. However when i run the angular application my API gives error as they cant find 

tables like dimensions.medium and datasets.diksha_contentqrcode_textbookdiksha0grade0subject0medium etc. 

I know that after uploading national programs CSV to the Minio Bucket i need to run the adapter service for the same.

My question is whether after running the adapter/nifi programs would i be able to see the tables like dimensions.medium and 

required datasets as mentioned earlier.?

**Comment by RNarmatha** [comment]
Before Uploading the data , need to verify whether all the columns are correct as per the cQube [template ](https://docs.google.com/spreadsheets/d/1sxnxv_YmxAxsN4blEkYyeE4SR2MD3A38SpmrKb-GNdw/edit?usp=sharing)
1. First upload the VSK schemas 
2. upload the rawdata
3. schedule the adaptor 
4. Run the latest code API as per the[ document ](https://docs.google.com/document/d/1xSTm90t_CsOqbsHc9QDzqhkhATjGzY66lz30F-LKHWo/edit?usp=sharing)
